math/libfaiss/Portfile

```diff
@@ -1,14 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
 PortGroup           compilers 1.0
+PortGroup           github 1.0
 
-github.setup        facebookresearch faiss 56383610bcb982d6591e2e2bea3516cb7723e04a
+github.setup        facebookresearch faiss 1.5.3 v
 name                libfaiss
-version             20180214
```

Comment:
I think there is an issue here with the versioning in the sense that `20180214` will be considered "newer" than `1.5.3.`. Please check and install the current version in `master` first, then change to your updated-libfaiss-branch, do a `portindex` and `port outdated`. If the `libfaiss` port doesn't show up there, this is indeed an issue and you should add `epoch 1`.

Reply:
The current `libfaiss` cannot be installed on current macOS because of a build error.

IS there a way of checking this issue without an install?

Reply:
I just went ahead and added `epoch 1`.

Reply:
> The current `libfaiss` cannot be installed on current macOS because of a build error.

Unless you are saying that no previous version of this port could ever be installed on any system—which I can see by looking at https://packages.macports.org/libfaiss/ is not the case—you should increase the epoch.

> I just went ahead and added `epoch 1`.

You added `epoch 0`, which is the default; to have any effect it would need to be `epoch 1`.

Reply:
Thanks done.